assets/aml-benchmark/components/benchmark-result-aggregator/spec.yaml

```diff
@@ -16,7 +16,14 @@ inputs:
     type: uri_folder
     description: The performance metrics in json format.
     optional: True
-
+  model_type:
+    type: string
+    optional: False
+    description: The endpoint model type.
+    enum:
+      - llama
+      - gpt
+    default: llama
```

Comment:
Did we forget to update the aggregator script to accommodate this?

Reply:
This is added by mistake, removed now.